draft-irtf-cfrg-frost.md

```diff
@@ -497,7 +497,9 @@ set of signing commitments for other signers in the participant list. Each signe
 MUST validate the inputs before processing the Coordinator's request. In particular,
 the Signer MUST validate commitment_list, deserializing each group Element in the
 list using DeserializeElement from {{dep-pog}}. If deserialization fails, the Signer
-MUST abort the protocol.
+MUST abort the protocol. Further, the signer MUST ensure that the message to be signed
+is in fact appropriate. However, doing so requires application-level logic, and so we
```

Comment:
Sorry, but I don't think my prior comment was clear. I don't think this is an appropriate MUST for the main protocol. It's not enforceable. This text belongs in the security considerations section.